### DISCUSSION TITLE
🔗 Support for YouTube Privacy-Mode in the Video Playlist widget

### DISCUSSION THREAD

**Discussion by mweber37** [elementor/elementor#16661] [Feature Request]
Hello,

In order to fullfill GDPR requirements for European websites, it would be extremely helpful if the video widgets (including the video playlist) would support YouTube's Privacy-Enhanced Mode.

This can be done easily by replacing the link "www.youtube.com" with "www.youtube-nocookie.com" within the youtube embeds. 

Many thanks
Michael

**Comment by m1ga** [comment]
I would like to see an option `click to show` so it won't even create a connection to the youtube server (and sending the IP without user interaction/GDPR consent).

**Comment by nicholaszein** [comment]
Hi,

The video already has the Privacy Mode since version 1.9.

However, I've changed the title to reflect the request for the same feature in the Video Playlist widget (Pro).

Regards.
